Utils/CommandHandler.py

```diff
@@ -0,0 +1,55 @@
+import IrcCommands
+
+#   When new command is created in IrcCommands folder, do the following
+#     1. Add the import in IrcCommands/__init__.py
+#     2. Create a function here to call the function in IrcCommands
+#     3. Add the command trigger to CommandsList
+#     4. Add trigger:function to CommandSwitch
+def Handle(user, msg, ircClient):
+    CommandsList = [
+        "is listening",
+        "is playing",
+        "!with",
+        "!mods",
+        "!discord",
+        "!help"
+    ]
+    CommandSwitch = {
+        "is listening" : isListening,
+        "is playing" : isPlaying,
+        "!with" : with_,
+        "!mods" : mods,
+        "!discord" : discord,
+        "!help" : help_,
+        "default" : default
+    }
+    actualCommand = parseCommand(msg, CommandsList)
+    commandToRun = CommandSwitch.get(actualCommand)
+    commandToRun(user, msg, ircClient)
```

Comment:
nitpicks just about python conventions here

- no function name should be uppercase (`Handle`)
- no variable name should be uppercase (`CommandsList`, `CommandsSwitch`)
  - python really should use `snake_case` but i guess we can use `camelCase` for consistency with sarah's code
- pls use `CommandSwitch[actualcommand]` instead of `CommandSwitch.get(actualComand)`